deps/make.blosc

```diff
@@ -19,6 +19,12 @@ OTHER_OBJ = bitshuffle-generic.o shuffle-generic.o shuffle.o blosclz.o blosc.o
 # Check whether the option -mavx2 is supported
 CHECK_AVX2 = $(shell if $(CC) $(CPPFLAGS) $(CFLAGS) -mavx2 -c -o shuffle-avx2.tmp shuffle-avx2.c; then echo 1; else echo 0; fi; rm -f shuffle-avx2.tmp)
 
+# include internal LZ4
+# there will only be one LZ4 dir
+LZ4_DIR = $(wildcard ../internal-complibs/lz4-*)
```

Comment:
I feel like if we can assume a gcc-compatible compiler we can assume GNU make (not that that's necessarily implied, but I feel it's a safe assumption).